pom.xml

```diff
@@ -38,8 +38,9 @@
     </parent>
 
     <properties>
-        <elasticsearch.version>1.0.0.RC1</elasticsearch.version>
-        <lucene.version>4.6.0</lucene.version>
+        <elasticsearch.version>2.0.0</elasticsearch.version>
+        <lucene.version>5.2.1</lucene.version>
+        <mvn.java.version>1.7</mvn.java.version>
```

Comment:
Took the liberty to upgrade to java 7.. it's been out for a while at least

As a side note, the project does not build with java8, due to much stricter javadoc requirements there. Opted to not change them in this PR.